build/common/installer/scripts/tomlparser-common-agent-config.rb

```diff
@@ -0,0 +1,109 @@
+#!/usr/local/bin/ruby
+
+#this should be require relative in Linux and require in windows, since it is a gem install on windows
```

Comment:
the os_type value is not being used to decide require type?

Reply:
this comment not applicable any more and cleaned up.